### DISCUSSION TITLE
Show visualizer without the player?

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [samhirtarif/react-audio-recorder#55] [Q&A] [answered]
I like the simplicity of the module, but the actual player won't work for me (aside from the weirdness of the "stop" button deleting and reseting). I really like the real-time visualizer most. Is there any way to use that visualizer separate from the controls?

[Here's an example](https://podinbox.com/buzzsprout):

<img width="686" alt="image" src="https://github.com/samhirtarif/react-audio-recorder/assets/154936/b7186fe6-27c9-4e4c-86d0-6ea29178736e">

**Comment by theDanielJLewis** [comment, marked as answer]
Well how about that? You actually already made the player available as a separate package:

https://www.npmjs.com/~samhirtarif

Thanks!

**Comment by samhirtarif** [comment]
Glad that you found the package already.
There is a separate package for the [Visualizer](https://www.npmjs.com/package/react-audio-visualize) and also a package to for an [audio player component](https://www.npmjs.com/package/react-audio-player-component) as well
